.circleci/config.yml

```diff
@@ -241,13 +237,37 @@ jobs:
   # This helps ensure the overall stability of the plugin prior to publishing.
   linux-tarball-test:
     <<: *defaults
+    parameters:
+      node-version: &node_version_param # Define an anchor here, since we'll want to use this parameter again elsewhere.
+        type: enum
+        enum:
+          - lts
+          - current
+        default: lts
+        description: Which version of node should be used.
     steps:
       - attach_workspace:
           at: ~/repo
 
-      # We'll want to use the LTS version of node
-      - update_node:
-          os: linux
+      # Install the appropriate version of node
+      - when:
+          condition:
+            equal:
+              - lts
+              - << parameters.node-version >>
+          steps:
+            - update_node:
+                os: linux
+                lts: true
+      - when:
+          condition:
+            equal:
+              - current
+              - << parameters.node-version >>
+          steps:
+            - update_node:
+                os: linux
+                lts: false
```

Comment:
The jobs have enum-type parameters, and the command has a boolean-type parameter. Yaml doesn't seem to support in-line boolean logic, so this construct is required.

Reply:
Same as above